ratcave/mixins.py

```diff
@@ -71,16 +73,29 @@ def __init__(self, position=(0., 0., 0.), rotation=(0., 0., 0.), scale=1., *args
             scale (float): uniform scale factor. 1 = no scaling.
         """
         super(Physical, self).__init__(*args, **kwargs)
-        self.x, self.y, self.z = position
-        self.rot_x, self.rot_y, self.rot_z = rotation
+        self.__dict__['x'], self.__dict__['y'], self.__dict__['z'] = position
```

Comment:
This is an implementation feature of the "observed" variables (__observables = ('x', 'y', 'z', 'rot_x', 'rot_y', 'rot_z', 'scale')). It avoids calling the __setattr__ method to assign initial values for them.